step-templates/iis-apppool-create.json

```diff
@@ -2,9 +2,9 @@
ersion -ParameterName 'Managed Runtime Version'\nValidate-Parameter -Parameter $appPoolManagedPipelineMode -ParameterName 'Managed Pipeline Mode'\nValidate-Parameter -Parameter $appPoolIdleTimeout -ParameterName 'Process Idle Timeout'\nValidate-Parameter -Parameter $appPoolPeriodicRecycleTime -ParameterName 'Periodic Recycle Time'\nValidate-Parameter -Parameter $appPoolMaxProcesses -ParameterName 'Maximum Worker Processes'\n\n$iis = (New-Object Microsoft.Web.Administration.ServerManager)\n$pool = $iis.ApplicationPools | Where-Object {$_.Name -eq $appPoolName} | Select-Object -First 1\nif ($pool -eq $null) {\n    Write-Output ('Creating Application Pool {0}' -f $appPoolName)\n    Execute-Retry {\n        $iis = (New-Object Microsoft.Web.Administration.ServerManager)\n        $iis.ApplicationPools.Add($appPoolName)\n        $iis.CommitChanges()\n    }\n}\nelse {\n    Write-Output ('Application Pool {0} already exists, reconfiguring ...' -f $appPoolName)\n}\n$list = Get-ScheduledTimes -Schedule $appPoolPeriodicRecycleTime\nExecute-Retry {\n    $iis = (New-Object Microsoft.Web.Administration.ServerManager)\n    $pool = $iis.ApplicationPools | Where-Object {$_.Name -eq $appPoolName} | Select-Object -First 1\n    Write-Output ('Setting: AutoStart = {0}' -f $appPoolAutoStart)\n    $pool.AutoStart = $appPoolAutoStart\n    Write-Output ('Setting: Enable32BitAppOnWin64 = {0}' -f $appPoolEnable32BitAppOnWin64)\n    $pool.Enable32BitAppOnWin64 = $appPoolEnable32BitAppOnWin64\n    Write-Output ('Setting: IdentityType = {0}' -f $appPoolIdentityType)\n    $pool.ProcessModel.IdentityType = $appPoolIdentityType\n    if ($appPoolIdentityType -eq 3) {\n        Write-Output ('Setting: UserName = {0}' -f $appPoolIdentityUser)\n        $pool.ProcessModel.UserName = $appPoolIdentityUser\n        Write-Output ('Setting: Password = [Omitted For Security]')\n        $pool.ProcessModel.Password = $appPoolIdentityPassword\n    }\n    Write-Output ('Setting: ManagedRuntimeVersion = {0}' -f $appPoolManagedRuntimeVersion)\n    $pool.ManagedRuntimeVersion = $appPoolManagedRuntimeVersion\n    Write-Output ('Setting: ManagedPipelineMode = {0}' -f $appPoolManagedPipelineMode)\n    $pool.ManagedPipelineMode = $appPoolManagedPipelineMode\n    Write-Output ('Setting: IdleTimeout = {0}' -f $appPoolIdleTimeout)\n    $pool.ProcessModel.IdleTimeout = $appPoolIdleTimeout\n    Write-Output ('Setting: MaxProcesses = {0}' -f $appPoolMaxProcesses)\n    $pool.ProcessModel.MaxProcesses = $appPoolMaxProcesses\n    Write-Output ('Setting: RegularTimeInterval = {0}' -f $appPoolRegularTimeInterval)\n    $pool.Recycling.PeriodicRestart.Time  = $appPoolRegularTimeInterval\n    Write-Output ('Setting: QueueLength = {0}' -f $appPoolQueueLength)\n    $pool.QueueLength  = $appPoolQueueLength\n    Write-Output ('Setting: Schedule = {0}' -f $appPoolPeriodicRecycleTime)\n    foreach($timestamp in $list) {\n        $pool.Recycling.PeriodicRestart.Schedule.Add($timestamp)\n    }\n    $iis.CommitChanges()\n}",
```

Comment:
I think the new property is in use here.

Reply:
My sincere apologies @pr0peller, not sure what I was doing when reviewing your PR but I must've clearly gone blind for a moment. I've checked the script again and you are correct, the parameter is fine and is being used correctly.

Again, my apologies for my erroneous review.